devel/fortio/http.go

```diff
@@ -644,15 +651,17 @@ func (c *BasicClient) readResponse(conn *net.TCPConn) {
 			}
 			break // we're done!
 		}
-	}
+	} // end of big for loop
 	// Figure out whether to keep or close the socket:
 	if keepAlive && c.code == http.StatusOK {
 		c.socket = conn // keep the open socket
 	} else {
```

Comment:
nit: since this appears to be the end of the method, we could just return inside of the conditional, and avoid the else.

similarly, in the contained if-else, we could just return inside the if and avoid the else altogether.

Reply:
that whole method has a big TODO refactor but I don't want to do that just before the release